### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by samccann**
##### SUMMARY

Attempting to bring all the ansible readme's in line...

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

docs
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,12 +74,25 @@ collections:
 - name: ansible.windows
 ```
 
+**NOTE**: For Ansible 2.9, you may not see deprecation warnings when you run your playbooks with this collection. Use this documentation to track when a module is deprecated.
```
Comment (jborean93) [inline README.md:77]: This collection won't work for 2.9, or at least the full collection doesn't. We are only targeting 2.10+.

Comment (samccann) [inline README.md:77]:
```suggestion
```

File: README.md [hunk 2]
```diff
@@ -114,7 +124,7 @@ You can run the collection's test suites with the commands:
 
 The current process for publishing new versions of the Windows Core Collection is manual, and requires a user who has access to the `ansible` namespace on Ansible Galaxy and Automation Hub to publish the build artifact.
 
-  1. Ensure `CHANGELOG.md` contains all the latest changes.
+  1. Regenerate `CHANGELOG.rst` with the [antsibull-changelog generator](https://github.com/ansible-community/antsibull-changelog).
```
Comment (jborean93) [inline README.md:127]: I just recently updated the README for this stuff in https://github.com/ansible-collections/ansible.windows/pull/70 which already has something like this. I copied whatever was on the community.crypto repo.

Comment (samccann) [reply README.md:127]: kewl thanks!

File: README.md [hunk 3]
```diff
@@ -115,12 +125,12 @@ You can run the collection's test suites with the commands:
...
 
   1. Update the CHANGELOG:
-    1. Make sure you have [`antsibull-changelog`](https://pypi.org/project/antsibull-changelog/) installed.
-    1. Make sure there are fragments for all known changes in `changelogs/fragments`.
-    1. Run `antsibull-changelog release`
-  1. Update `galaxy.yml` with the new `version` for the collection.
-  1. Create a release in GitHub to tag the commit at the version to build.
-  1. Run the following commands to build and release the new version on Galaxy:
+    1. Make sure you have [`antsibull-changelog`](https://pypi.org/project/antsibull-changelog/) installed.  
+    2. Make sure there are fragments for all known changes in `changelogs/fragments`.
```
Comment (jborean93) [inline README.md:129]: Isn't it simpler to keep them at 1. Makes it easier to add/change/remove entries without re-ordering the entire list.

File: README.md [hunk 4]
```diff
@@ -115,12 +125,12 @@ You can run the collection's test suites with the commands:
...
   1. Update the CHANGELOG:
-    1. Make sure you have [`antsibull-changelog`](https://pypi.org/project/antsibull-changelog/) installed.
-    1. Make sure there are fragments for all known changes in `changelogs/fragments`.
-    1. Run `antsibull-changelog release`
-  1. Update `galaxy.yml` with the new `version` for the collection.
-  1. Create a release in GitHub to tag the commit at the version to build.
-  1. Run the following commands to build and release the new version on Galaxy:
+    1. Make sure you have [`antsibull-changelog`](https://pypi.org/project/antsibull-changelog/) installed.  
+    2. Make sure there are fragments for all known changes in `changelogs/fragments`.  
+    3. Run `antsibull-changelog release`.
```
Comment (jborean93) [inline README.md:130]: Some whitespace has been added to the end of this line and some other above it.

File: README.md (hunk 3 above)
Comment (samccann) [reply README.md:129]: 1, 1. 1. doesn't make sense from a docs perspective. If you feel strongly that you don't want to deal with renumbering, we should use bullet items instead. (tho I can't promise I won't forget 3 months from now and try to number 'em again.. it's a docs thing :-)

Comment (jborean93) [reply README.md:129]: The raw markdown shows 1. 1. 1. but the actual markdown output orders them properly. Sounds like a no-brainer to have something do it automatically for you.

Comment (jborean93) [reply README.md:129]: But I'm also happy them being dot points, I just copied what was from the crypto README.

Comment (samccann) [reply README.md]: something going wonky on my side. gonna back out some commits so bear with a moment or two

**Comment by samccann** [comment]
Closing in favor of https://github.com/ansible-collections/ansible.windows/pull/76 of a fresh pull, cuz... yeah.. makin' a mess here.

FWIW - local markdown preview (atom editor) doesn't match github markdown view and I worry what will be in Galaxy. 
The 1. 1. autonumbering works fine for simple list:
```
1. bla
1. foo
1. bar.
```
displays as:

1. bla
1. foo
1. bar.

But when I put in substeps, it just wasn't working in github view for me.
```
1. bla
  1. foobla
  1. foo bla bla
1. foo
```

displays as:

1. bla
  1. foobla
  1. foo bla bla
1. foo

**Comment by jborean93** [comment]
Yea in the end I ended up doing the bullet points instead because the nested 1.s weren’t working. Sorry for all the merge conflicts as I’ve been trying to get all this ready around the config and sanity stuff. It should be free for any changes now as I did the last work for that yesterday.

**Pull request closed without merging**